protocol/protobuf/community_update.proto

```diff
@@ -36,6 +37,7 @@ message CommunityEvent {
     COMMUNITY_MEMBER_KICK = 14;
     COMMUNITY_MEMBER_BAN = 15;
     COMMUNITY_MEMBER_UNBAN = 16;
+    COMMUNITY_TOKEN_ADD = 17;
```

Comment:
Can TokenMasters and Owners only add tokens? Can't they remove the token?

Reply:
Not in the current design. Also, the token cannot be deleted from the blockchain, so it would only unlink from the community.